backend/src/main/java/org/nexters/cultureland/api/culture/service/CultureServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package org.nexters.cultureland.api.culture.service;
+
+import org.modelmapper.ModelMapper;
+import org.nexters.cultureland.api.culture.dto.CultureDto;
+import org.nexters.cultureland.api.culture.repo.CultureRawRepo;
+import org.nexters.cultureland.api.culture.repo.CultureRepo;
+import org.nexters.cultureland.api.culture.model.Culture;
+import org.nexters.cultureland.api.culture.model.CultureRawData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service("CultureService")
+@Transactional
+public class CultureServiceImpl implements CultureService{
+
+    @Autowired
+    private  ModelMapper modelMapper;
+
+    @Autowired
+    private CultureRepo cultureRepo;
+
+    @Autowired
+    private CultureRawRepo cultureRawRepo;
+
+    //전체 목록 조회
+    public List getList() {
+        return cultureRawRepo.findList();
+    }
+
+
+    public List getByCategory(String category) {
+        //return cultureRepo.findByCultureName(category).stream().map(culture -> modelMapper.map(culture,CultureDto.class)).collect(Collectors.toList());
+        return cultureRepo.findByCultureName(category);
+    }
+
+
+    public Optional<CultureRawData> getByCultureId(Long id) {
+        return cultureRawRepo.findById(id);
```

Comment:
Optional을 바로 리턴해버리면 Controller에서 없을 경우 Exception 처리를 해줘야하니깐 여기서 Exception 처리까지 하는거는 어떄??

```suggestion
        return cultureRawRepo.findById(id)
                .orElseThrow(() -> new CustomException("message"));
```

Reply:
오호 예외처리 부분 수정할게요! 고마워!